<?xml version="1.0" ?>
<launch>
	<node pkg="bobac2_joy" type="bobac2_joy_node" name="bobac2_joy" >
	<param name="axis_linear_x" value="1" type="int" />
	<param name="axis_linear_y" value="0" type="int" />
	<param name="axis_angular" value="3" type="int" />
	<param name="linear_max" value="0.5" type="double" />
	<param name="angular_max" value="2" type="double" />
	</node>
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node">
	<param name="dev" type="string" value="/dev/input/js0" />
    	<param name="deadzone" value="0.12" />
	</node>
</launch>
